.evergreen/evergreen.yml.in

```diff
@@ -919,6 +919,8 @@ functions:
           source .evergreen/setup-env.sh
           export PUPPETEER_SKIP_DOWNLOAD="true"
           npm run evergreen-release draft
+          git add .
```

Comment:
It's surprising that this file has changed, but .evergreen.yml hasn't.

Reply:
ah, it's because this generation needs to be run manually; forgot to do that